packages/client/setupTests.js

```diff
@@ -0,0 +1,8 @@
+//setupTests.tsx
+const nodeCrypto = require("crypto");
+
+globalThis.crypto = {
```

Comment:
can you leave a comment with a link to the Stackoverflow page where you found this?

Reply:
https://stackoverflow.com/a/62601487/1194694

This one with one slight modification (globalThis instead of window) to make it runtime agnostic